docs/user-guide/extending-cobbler.rst

```diff
@@ -0,0 +1,209 @@
+*****************
+Extending Cobbler
+*****************
+
+This section covers methods to extend the functionality of Cobbler through the use of :ref:`triggers` and
+:ref:`modules`, as well as through extensions to the Cheetah templating system.
+
+.. _triggers:
+
+Triggers
+########
+
+About
+=====
+
+Cobbler triggers provide a way to tie user-defined actions to certain Cobbler commands -- for instance, to provide
+additional logging, integration with apps like Puppet or cfengine, set up SSH keys, tieing in with a DNS server
+configuration script, or for some other purpose.
+
+Cobbler Triggers should be Python modules written using the low-level Python API for maximum speed, but could also be
```

Comment:
```suggestion
Cobbler triggers should be Python modules using the low-level Python API for maximum speed, but could also be
```

Reply:
That is indeed correct since there are "Cobbler Triggers".

Reply:
Ah, I did not know that. The word `written` still sounds strange in this context.